Octokit.Tests.Integration/Clients/OrganizationOutsideCollaboratorsClientTests.cs

```diff
@@ -182,7 +182,7 @@ public TheGetAllMethod()
                 using (var context = await _gitHub.CreateRepositoryContext(Helper.Organization, new NewRepository(repoName)))
                 {
                     await _gitHub.Repository.Collaborator.Add(context.RepositoryOwner, context.RepositoryName, _fixtureCollaborator);
-                    await _gitHub.Repository.Collaborator.Add(context.RepositoryOwner, context.RepositoryName, "alfhenrik");
+                    await _gitHub.Repository.Collaborator.Add(context.RepositoryOwner, context.RepositoryName, "alfhenrik-test-2");
```

Comment:
Ah ok... I could enable 2FA on one of the octokit test accounts if you think that's a better idea?